0000-interledger-protocol-4/0000-interledger-protocol-4.md

```diff
@@ -0,0 +1,159 @@
+
+ILPv4 will probably be used with a Transport Layer protocol such as the [Pre-Shared Key V2 (PSKv2)](../0025-pre-shared-key-2/0025-pre-shared-key-2.md) protocol, which handles the generation of ILP conditions and fulfillments.
+
+### Differences from Previous Versions of ILP
+
+- **Designed for Smaller, More Homogenous Packet Amounts** - ILPv4 applies a number of simplifications based on the change from supporting all packet amounts to focusing the Interledger layer on lower-value packets. Larger amounts can be sent using chunked payments, but the core network is optimized for sending large volumes of small packets. This renders unnecessary some of the more complex features of previous versions like Liquidity Curves for expressing how exchange rates may vary depending on the packet amount. Lower-value packets also help to minimize [connector risks](../0018-connector-risk-mitigations/0018-connector-risk-mitigations.md).
+- **Payment Channels, Not On-Ledger Escrow** - Since ILPv4 is optimized for smaller packets, speed and cost are of greater importance than in previous versions. ILPv4 uses ledgers or payment channels for settling bilateral payment obligations but ILPv4 packets are sent just between connectors, rather than through the underlying ledgers themselves. This enables packet timeouts to be short, because they do not need to include the processing time of slower ledgers, which further reduces [connector risks](../0018-connector-risk-mitigations/0018-connector-risk-mitigations.md).
+- **Forwarding, Not Delivery** - ILPv4 connectors forward packets based on their local exchange rates, in constrast with the first version of ILP in which connectors would attempt to deliver a fixed destination amount. Now, instead of fixed destination amount delivery being built into the core protocol, senders may use higher-level protocols to indicate the minimum amount the receiver should accept for a given packet and receivers can reject packets with less than that. This significantly simplifies the connector behavior, because the do not need to maintain up-to-date price information on the entire rest of the network and can simply apply their local rate instead. If receivers want to receive no more than a certain amount, they can reject packets that go too far over the amount and senders can retry the packet with lower amounts.
+- **Quoting is an Application Concern** - Connectors are only responsible for forwarding ILP packets and do not need to implement a [separate protocol for quoting](../0008-interledger-quoting-protocol/0008-interledger-quoting-protocol.md). Applications can use test packets to determine the exchange rate of a particular path.
+
+## Flow
+
+Before the protocol starts, the sender and receiver use a higher-level protocol to agree on the destination account and packet condition.
```

Comment:
We should say that this is just the flow of a single ILP packet

Reply:
@sharafian this is the setup protocol

Reply:
I mean the subsequent steps in the flow section define the flow of a single ILP packet, which may not be immediately clear (that this will be repeated many times in the flow of a payment)